src/main/archetype/ui.config/src/main/content/jcr_root/apps/__appId__/osgiconfig/config.prod/org.apache.sling.commons.log.LogManager.factory.config~__appId__.cfg.json

```diff
@@ -3,7 +3,7 @@
   "org.apache.sling.commons.log.names": [
     "${package}"
   ],
-  "org.apache.sling.commons.log.level": "error",
+  "org.apache.sling.commons.log.level": "WARN",
```

Comment:
Shouldn't this be ERROR?

Reply:
You are right, as these are removed on cloud builds.